### PR TITLE
[loss] Combined cross entropy with sigmoid @open sesame 06/22 19:54

### PR DESCRIPTION
**Pull request by kparichay**
Combined cross-entropy with sigmoid version for loss because of its higher stability
Note that this happens internally and is not exposed outside

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

**Comment by kparichay** [comment]
rebased.
